pkg/server/v1/consts.go

```diff
@@ -22,15 +22,17 @@ const (
 	KubescapeFallbackCustomerGUID = "11111111-1111-1111-1111-111111111111"
 
 	// URL query parameters
-	QueryParamCustomerGUID  = "customerGUID"
-	QueryParamScope         = "scope"
-	QueryParamFrameworkName = "frameworkName"
-	QueryParamPolicyName    = "policyName"
-	QueryParamClusterName   = "clusterName"
-	QueryParamContextName   = "contextName"
-	QueryParamReport        = "reportGUID"
-	QueryParamJobID         = "jobID"
-	QueryParamRegistryName  = "registryName"
+	QueryParamCustomerGUID        = "customerGUID"
+	QueryParamScope               = "scope"
+	QueryParamFrameworkName       = "frameworkName"
+	QueryParamPolicyName          = "policyName"
+	QueryParamClusterName         = "clusterName"
+	QueryParamContextName         = "contextName"
+	QueryParamReport              = "reportGUID"
+	QueryParamJobID               = "jobID"
+	QueryParamRegistryName        = "registryName"
+	QueryParamGitRegoStoreVersion = "gitRegoStoreVersion"
+	RegolibraryVersionV2          = "v2"
```

Comment:
do you think it's a good idea to put `v2` in the name? how do you upgrade to v3?

Reply:
good point. I'll change it to `RegolibraryVersion`